### PR TITLE
chore(ui-templates): new templates branding for v4

### LINKED ISSUE #26959: Refreshed NuxtWelcome design (+ other error pages?)

### PR DESCRIPTION
**Pull request by atinux**
Resolves #26959

CleanShot 2024-06-26 at 15 23 25@2x
CleanShot 2024-06-26 at 15 23 37@2x
CleanShot 2024-06-26 at 15 29 26@2x

https://github.com/nuxt/nuxt/assets/904724/6110a182-dbe2-4948-9101-f57c4d6ad4e6

https://github.com/nuxt/nuxt/assets/904724/8a8d269a-b480-4874-9b2f-58f132a3657c

I know it does not resolve all the comments from the issue, but this is a good start IMO for future iterations.

### CODE REVIEW

**Comment by Atinux** [comment]
We need a pay to get the version (major + minor) from the package.json somehow @danielroe, I let you take a look at it

File: packages/ui-templates/templates/loading/index.html [hunk 1]
```diff
@@ -7,166 +7,58 @@
...
-
-      animationToggle.addEventListener('click', () => toggleAnimation(), { passive: true})
-      body.addEventListener('mousemove', onMouseMove, { passive: true })
-      body.addEventListener('mouseleave', onFocusOut, { passive: true })
-
-      toggleAnimation(enableAnimation)
-
       if (typeof window.fetch === 'undefined') {
-        setTimeout(() => window.location.reload(), 1000)
+        setTimeout(() => window.location.reload(), 200)
```
Comment (danielroe) [inline packages/ui-templates/templates/loading/index.html:61]: I've reduced the reload amount to 200ms - seem ok?

**Review by danielroe — APPROVED** [review]
😍

**Comment by gvozdenkov** [comment]
@atinux Why you use 2 separate grid insted 1 grid with ul > li clear semantic list of links?
image

**Comment by danielroe** [comment]
improvements welcome! 🙏

**Comment by atinux** [comment]
Agree, feel free to make a PR if you find better!

**Pull request merged**